root/treeformula/retobj/runretobjTest.C

```diff
@@ -7,8 +7,8 @@ gSystem->Load("libTreePlayer");
 gROOT->ProcessLine(".L helper.C");
 #endif
 
-gSystem->Load("libEvent");
-TFile *Event = TFile::Open("Event.new.split0.root");
+gSystem->Load("libTreeFormulaRetobjGeneration");
+TFile *Event = TFile::Open("Event.root");
```

Comment:
This is a very common file produced by roottest, it is much better to clarify which configuration of the file we are using.

Reply:
Hi @pcanal, thank you very much for the comment! What do you mean exactly by "to clarify which configuration of the file we are using"?

Reply:
See `$ROOTSYS/test/MainEvent.cxx`.  For whatever reason this example use the configuration of the typical Event file with split level 0, and 'new' style branches, let's not lose that information (and not change the configuration of the generated file either)